src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -109,6 +109,10 @@ SYSTEM_DTB_ADDR="0x40000"
 KERNEL_ADDR="0x20100000"
 ROOTFS_ADDR="0x21000000"
 
+# this address needs to be in sync with VMR
```

Comment:
Could you add a comment to where one can find it in the VMR.   

My concern is that if someone changes it in the VMR how would this script know.   Is it possible for this date to be data driving by a JSON or some other file.

Reply:
This is temporary solution. We will change it to data driven after xsabin section metadata is supported.